distributed/utils_test.py

```diff
@@ -1198,23 +1198,6 @@ def popen(args, **kwargs):
                 print(out.decode())
 
 
-def wait_for_port(address, timeout=5):
```

Comment:
I have no idea if anybody is using this but imho this should not be part of our public API